assets/translations/en-us.yaml

```diff
@@ -3842,7 +3843,7 @@ setup:
     dockerSuffix: ""
     helmPrefix: 'For a Helm installation, run:'
     helmSuffix: ""
-  eula: I agree to the <a href="https://rancher.com/eula" target="_blank" rel="noopener noreferrer nofollow">terms and conditions</a> for using Rancher.
+  eula: I agree to the <a href="https://rancher.com/eula" target="_blank" rel="noopener noreferrer nofollow">terms and conditions</a> for using {name}.
```

Comment:
I thought about suggesting the `{vendor}` tag instead (it's what's shown in the log in screen `Welcome to <x>`). However the T&Cs are strictly Rancher/Rancher product related and isn't customisable. So either Rancher/Harvester is good